.eslintrc.yml

```diff
@@ -0,0 +1,22 @@
+env:
+  browser: true
+  es6: true
+  node: true
+
+extends:
+  - eslint:recommended
+
+root: true
+
+rules:
+  comma-dangle: off
+  eol-last: error
+  eqeqeq: error
+  handle-callback-err: error
+  indent: [error, 2]
+  no-console: off
+  no-multi-spaces: error
+  no-unused-vars: [error, {vars: all, args: none}]
+  one-var: [error, never]
+  quotes: [error, double]
+  semi: [error, always]
```

Comment:
Added rules from https://github.com/gregglind/shield-study-cli/blob/master/.eslintrc.yml
Yeah, eventually we should make a shared rules config.